drivers/sensor/adxl345/adxl345.c

```diff
@@ -255,34 +353,41 @@ static int adxl345_init(const struct device *dev)
+	.fifo_mode = DT_INST_PROP(inst, fifo),
+
+#define ADXL345_CONFIG_SPI(inst)                                                                   \
+	{                                                                                          \
+		.bus = {.spi = SPI_DT_SPEC_INST_GET(inst,                                          \
+						    SPI_WORD_SET(8) | SPI_TRANSFER_MSB |           \
+							    SPI_MODE_CPOL | SPI_MODE_CPHA,         \
+						    0)},                                           \
+		.bus_is_ready = adxl345_bus_is_ready_spi, .reg_access = adxl345_reg_access_spi,    \
+		ADXL345_CONFIG(inst)                                                               \
 	}
 
-#define ADXL345_CONFIG_I2C(inst)			    \
-	{						    \
-		.bus = {.i2c = I2C_DT_SPEC_INST_GET(inst)}, \
-		.bus_is_ready = adxl345_bus_is_ready_i2c,   \
-		.reg_access = adxl345_reg_access_i2c,	    \
+#define ADXL345_CONFIG_I2C(inst)                                                                   \
+	{                                                                                          \
+		.bus = {.i2c = I2C_DT_SPEC_INST_GET(inst)},                                        \
+		.bus_is_ready = adxl345_bus_is_ready_i2c, .reg_access = adxl345_reg_access_i2c,    \
+		ADXL345_CONFIG(inst)                                                               \
 	}
 
-#define ADXL345_DEFINE(inst)								\
-	static struct adxl345_dev_data adxl345_data_##inst;				\
-											\
-	static const struct adxl345_dev_config adxl345_config_##inst =                  \
-		COND_CODE_1(DT_INST_ON_BUS(inst, spi), (ADXL345_CONFIG_SPI(inst)),      \
-			    (ADXL345_CONFIG_I2C(inst)));                                \
-                                                                                        \
-	SENSOR_DEVICE_DT_INST_DEFINE(inst, adxl345_init, NULL,				\
-			      &adxl345_data_##inst, &adxl345_config_##inst, POST_KERNEL,\
-			      CONFIG_SENSOR_INIT_PRIORITY, &adxl345_api_funcs);		\
-
-DT_INST_FOREACH_STATUS_OKAY(ADXL345_DEFINE)
+#define ADXL345_DEFINE(inst)                                                                       \
+	static struct adxl345_dev_data adxl345_data_##inst;                                        \
+                                                                                                   \
+	static const struct adxl345_dev_config adxl345_config_##inst =                             \
+		COND_CODE_1(DT_INST_ON_BUS(inst, spi), (ADXL345_CONFIG_SPI(inst)),                 \
+			    (ADXL345_CONFIG_I2C(inst)));                                           \
+                                                                                                   \
+	SENSOR_DEVICE_DT_INST_DEFINE(inst, adxl345_init, NULL, &adxl345_data_##inst,               \
+				     &adxl345_config_##inst, POST_KERNEL,                          \
+				     CONFIG_SENSOR_INIT_PRIORITY, &adxl345_api_funcs);
+
+DT_INST_FOREACH_STATUS_OKAY(ADXL345_DEFINE)
```

Comment:
This looks like a lot of whitespace noise. Please keep major whitespace reformatting changes in separate commits from functional changes

Reply:
Refactored code to reduce whitespace noise.